scripts/service-worker.js

```diff
@@ -0,0 +1,46 @@
+/* global importScripts, workbox */
+
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/3.6.1/workbox-sw.js')
+importScripts('/scripts/precache.js')
+
+if (workbox) {
```

Comment:
Should this be `typeof workbox !== `undefined``?

Reply:
Both work, will update for clarity